qemu-lite/debian.rules-template

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/make -f
+include /usr/share/dpkg/default.mk
+export LANG=C
+%:
+	dh $@
+
+override_dh_auto_configure:
+	./configure \
```

Comment:
This should be calling https://github.com/kata-containers/packaging/blob/master/scripts/configure-hypervisor.sh now.